object_detection/core/preprocessor_test.py

```diff
@@ -15,7 +15,11 @@
 
 """Tests for object_detection.core.preprocessor."""
 
-import mock
```

Comment:
Please use the following order for imports:

import numpy as np
import six

import tensorflow as tf

from google3.third_party.tensorflow_models.object_detection.core import preprocessor
from google3.third_party.tensorflow_models.object_detection.core import standard_fields as fields

if six.PY2:
  import mock  # pylint: disable=g-import-not-at-top
else:
  from unittest import mock  # pylint: disable=g-import-not-at-top